.github/workflows/update-xla-commit-hash.yml

```
@@ -20,38 +20,59 @@ jobs:
         with:
           fetch-depth: 1
           submodules: false
+          token: ${{ secrets.MERGEBOT_TOKEN }}
```

Hmm, can we use a github token instead?
```suggestion
          token: ${{ secrets.GITHUB_TOKEN }}
```

---

if you push using the github token, it doesnt retrigger workflows
https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#triggering-a-workflow-from-a-workflow